browser/js/global.js

```diff
@@ -281,6 +281,113 @@ window.app = {
 		lang: navigatorLang
 	};
 
+	global.prefs = {
```

Comment:
I think currently for globals we use:
docstate.js and docstatefunctions.js which are available under `app.something`
But it's fine for now. Later we will rework them to TS and probably merge anyway.

Reply:
Alright

I imagine if we decide to do that, we'll need to add it to [the app object at the top of global.js](https://github.com/CollaboraOnline/online/blob/e4d2095c71f5f8ee861d36c3d51f0bde7de5e75b/browser/js/global.js#L8) anyway, since as it is used in initialization of other globals